index.html

```diff
@@ -4,7 +4,599 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="./css/index.css" />
+    <link rel="stylesheet" href=" https://unicons.iconscout.com/release/v3.0.6/css/line.css" />
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav class="nav d-flex j-space-between">
+        <div class="polygons d-flex">
+          <div class="polygon bg-yellow"></div>
+          <div class="triangle"></div>
+        </div>
+        <div class="menu mr-10">
+          <div class="rectangle"></div>
+          <div class="rectangle"></div>
+          <div class="rectangle"></div>
+        </div>
+      </nav>
+
+      <h1 class="header_name">Popoola Aishat</h1>
+      <p class="p_text">
+        Hello! I am a software developer! I can help you build a product, feature or website. Taka a
+        look of my works. If you like what you see and have a project you need coded, don't hesitate
+        and contact me.
+      </p>
+      <button type="button" class="collab-btn">Start Collaboration</button>
+      <div class="box-design-section d-flex">
+        <!-- first box -->
+        <div class="first-box d-flex">
+          <div class="square-box">
+            <div class="rhombus-1 d-flex j-center bg-yellow">
+              <div class="rhombus-2"></div>
+            </div>
+          </div>
+          <div class="square-box-2 d-flex j-center">
+            <div class="round"></div>
+          </div>
+          <div class="square-box-3 d-flex">
+            <div class="triangle-2"></div>
+          </div>
+          <div class="square-box-4">
+            <div class="semicircle d-flex j-center bg-yellow">
+              <div class="semicircle-chat"></div>
+            </div>
+          </div>
+        </div>
+        <!-- second box -->
+        <div class="second-box d-flex">
+          <div class="semicircle-2">
+            <div class="first-semi-circle h-half bg-yellow"></div>
+            <div class="second-semi-circle h-half bg-yellow"></div>
+          </div>
+          <div class="basis-half">
+            <div class="square-box-5">
+              <div class="triangle-3"></div>
+            </div>
+            <div class="square-box-6 ">
+              <div class="semicircle-3 d-flex j-center bg-yellow">
+                <div class="semicircle-chat-2"></div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- third box -->
+      </div>
```

Comment:
All this can be enclosed in a `section` to make it more semantic

Reply:
Noted sir.